.editorconfig

```diff
@@ -28,5 +33,5 @@ indent_size = unset
 indent_style = unset
 
 # ignore python
```

Comment:
this line should mention markdown files too

Reply:
This is the same as the file in rnaseq- there's the readme.md above?